build.go

```diff
@@ -37,6 +37,7 @@ var (
 	libc    string
 	pkgArch string
 	version string = "v1"
+	buildTags []string = []string{}
```

Comment:
The default value, a nil slice, is actually good enough. `doBuild` handles nil slices just like empty ones, which is the idiomatic way.
```suggestion
	buildTags []string
```